include/data.h

```diff
@@ -112,6 +112,8 @@ struct TypeInfo
     u16 damageCategory:2;    // Used for B_PHYSICAL_SPECIAL_SPLIT <= GEN_3
     u16 padding:14;
     const u32 *const paletteTMHM;
+    bool8 useSecondTypeIconPalette:1;
+    bool8 isSpecialCaseType:1;
```

Comment:
there is some padding that can be used

Reply:
it's not clear to me what change I need to make here

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5131/commits/25a25c3a02607af6f240df976771c52702aeeb41